components/admin_console/admin_sidebar_header/admin_sidebar_header.tsx

```diff
@@ -49,7 +49,7 @@ export default class SidebarHeader extends React.PureComponent<Props> {
                         </div>
                         <div className='user__name overflow--ellipsis whitespace--nowrap'>{'@' + me.username}</div>
                     </div>
-                    <button className='style--none'>
+                    <button type='button' className='style--none'>
```

Comment:
Non-blocking change but since we are touching up this portion, lets fix styling to match others.
```suggestion
                    <button 
                         type='button' 
                         className='style--none'
                     >
```